Notes/Algorithms/Data Structures/Hash-Tables.c

```diff
@@ -4,34 +4,231 @@
 
 #define SUCCESS 0
 #define ERROR -1
-#define OVERFLOW -2
-#define EMPTY -3
+
+#define TEST 9
```

Comment:
If you decide to keep these global variables, it would be helpful to add a short comment about what they signify and what they are used for.

Reply:
Updated in [ commit 7b759b3c93b27fb6aebfae99f124f8559fcf28af ]( https://github.com/kcarrel/Training-Notes/pull/12/commits/7b759b3c93b27fb6aebfae99f124f8559fcf28af) to begin moving away from global variables!